dask/dataframe/tests/test_indexing.py

```diff
@@ -410,6 +410,41 @@ def test_to_series():
     assert_eq(df.index.to_series(), ddf.index.to_series())
 
 
+def test_to_frame():
```

Comment:
Pandas `Index.to_frame` was added in the 0.21.0 release (see the [changelog entry](https://pandas.pydata.org/pandas-docs/stable/whatsnew/v0.21.0.html#new-functions-or-methods)). Since our minimum supported version of pandas is 0.19.0, this test is failing when we test against pandas versions < 0.21.0 were `pd.Index.to_frame` doesn't exist. We should probably raise a `NotImplementedError` for this case